src/controllers/handlers/lists-handlers.ts

```diff
@@ -43,3 +45,97 @@ export async function getPicksByListIdHandler(
     },
   }
 }
+
+export async function createPickInListHandler(
+  context: Pick<HandlerContextWithPath<"lists", "/v1/lists/:id/picks">, "components" | "params" | "request"> &
+    authorizationMiddleware.DecentralandSignatureContext
+): Promise<HTTPResponse<TPick>> {
+  const {
+    components: { lists },
+    verification,
+    params,
+    request,
+  } = context
+  const userAddress: string | undefined = verification?.auth.toLowerCase()
+  let body: { itemId?: string }
+
+  if (!userAddress) {
+    return {
+      status: StatusCode.UNAUTHORIZED,
+      body: {
+        ok: false,
+        message: "Unauthorized",
+      },
+    }
+  }
+
+  try {
+    body = await request.json()
+    if (!body.itemId || (body.itemId && typeof body.itemId !== "string")) {
+      return {
+        status: StatusCode.BAD_REQUEST,
+        body: {
+          ok: false,
+          message: "The property itemId is missing or is not of string type.",
+        },
+      }
+    }
+  } catch (error) {
+    return {
+      status: StatusCode.BAD_REQUEST,
+      body: {
+        ok: false,
+        message: "The body must contain a parsable JSON.",
+      },
+    }
+  }
+
+  try {
+    const picksByListIdResult = await lists.addPickToList(params.id, body.itemId, userAddress)
```

Comment:
```suggestion
    const addPickToListResult = await lists.addPickToList(params.id, body.itemId, userAddress)
```

just because picksByList seem to be an array

Reply:
A copy and paste issue 🤦